### PR TITLE
[MOD-2322] Android: Parity for mapclick event

### PR DESCRIPTION
**Pull request by m1ga**
JIRA: https://jira.appcelerator.org/browse/MOD-2322

### CODE REVIEW

File: android/src/ti/map/TiUIMapView.java [hunk 1]
```diff
@@ -865,6 +865,10 @@ public void onMapClick(LatLng point) {
 		}
 		clickablePolylines.clear();
 
+		KrollDict d = new KrollDict();
+		d.put(TiC.PROPERTY_LATITUDE, point.latitude);
+		d.put(TiC.PROPERTY_LONGITUDE, point.longitude);
+		proxy.fireEvent("mapclick", d);
```
Comment (hansemannn) [inline android/src/ti/map/TiUIMapView.java]: iOS also has the current instance / self inside the `map` key. For 100 % parity, that would be nice. Also nice would be to check if the event listener exists prior to firing the event. Not sure if Titanium Android can do that though.

Comment (m1ga) [inline android/src/ti/map/TiUIMapView.java]: this is the whole output: https://gist.github.com/m1ga/2a5ff0409d295f4eea84155df3e307c4

Comment (hansemannn) [inline android/src/ti/map/TiUIMapView.java]: Ok, so `source` already includes it. I think other iOS events have the `map` one as well, wondering if Android has it there.

**Comment by m1ga** [comment]
The "map" property is there so I've put it in and used constants now

**Comment by m1ga** [comment]
Please check the documentation

**Comment by hansemannn** [comment]
LGTM, awesome!

**Pull request merged**